### DISCUSSION TITLE
FIL-RetroPGF Round 2 - Solana ledger applications

### DISCUSSION THREAD

**Discussion by anjor** [filecoin-project/community#721] [RetroPGF]
There have been 3 separate applications regarding the solana ledger archival work for the retro-pgf round 2. This discussion is an attempt at disambiguating the 3.

1) Old Faithful - Old faithful project by Triton designed the IPLD schema for the solana blockchain, handled the devops required to prepare and host car files, as well as handled the coordination across multiple different SPs to onboard the data to filecoin. The actual sealing and indexing on the filecoin side was done by the SPs. Triton pays the SPs out of band in fiat currency for this work.
2) Old Faithful <> Curio - DCent has been the first SP partner to Triton in this onboarding process, and had to make tweaks to the SP-side stack to make the onboarding process seamless. I will let DCent folks chime in.
3) Molly's application - FF helped with marketing at events around breakpoint.

**Comment by protocolin** [comment]
Helpful! Is this the link for (3) above? 

https://www.fil-retropgf.io/projects/0x646570350cbe29b1926287644ecdd48bad242a1373b8e7e18badea3e5ddb85e7
